### PR TITLE
OP: add ConvTranspose

### PR DESCRIPTION
**Pull request by IkbeomJeon**
- Tranpose convolution
see decription : https://github.com/onnx/onnx/blob/master/docs/Operators.md#ConvTranspose

- Sigmoid
It is refered to 'Logistic' in your Operation Code. I understand it is sigmoid fuction. see : https://github.com/onnx/onnx/blob/master/docs/Operators.md#Sigmoid

### CODE REVIEW

**Comment by IkbeomJeon** [comment]
and Thanks your contributions.  :)

File: tests/test_models.py [hunk 1]
```diff
@@ -38,6 +38,8 @@ def test_ops_implicit_layout():
         'fullyconnected.float32',
         'fullyconnected-relu6.float32',
         'maxpooling.float32',
+        'conv-transpose.float32',
+        'conv-transpose_relu.float32',
```
Comment (jackwish) [inline tests/test_models.py:42]: I typically name the files with `-` as the delimiter, i.e. `op-op-op.<dtype>`

File: tests/test_models.py [hunk 2]
```diff
@@ -82,6 +84,8 @@ def test_ops_layout_transparent():
 def test_networks():
     NETWORK_LIST = (
         'mobilenet_v1_0.25_128',
+        'object_detection_3d_sneakers',
```
Comment (jackwish) [inline tests/test_models.py:87]: In most cases, I'd like to keep the `master` branch small enough. That means, we keep only test purpose operator TFLite model on `master` branch, and track more network-wide models in [`more-model-test`](https://github.com/jackwish/tflite2onnx/tree/more-model-test) branch. I can help to move this model to that branch after this PR got merged.

File: tests/test_models.py [hunk 3]
```diff
@@ -82,6 +84,8 @@ def test_ops_layout_transparent():
...
+
```
Comment (jackwish) [inline tests/test_models.py:88]: Maybe remove this blank line?

File: tflite2onnx/op/conv_transpose.py [hunk 4]
```diff
@@ -0,0 +1,84 @@
+import logging
```
Comment (jackwish) [inline tflite2onnx/op/conv_transpose.py:1]: What about including the `ConvTranspose` class in `tflite2onnx/op/conv.py` as it's an altered version of Conv somehow. Maybe we can even share to utils just like [Pooling](https://github.com/jackwish/tflite2onnx/blob/master/tflite2onnx/op/pooling.py).

File: tflite2onnx/op/logistic.py [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+        self.parseInput(0)
+        self.parseOutput(0)
+
+        self.setParsed()
+
+    def propagatableTensors(self):
+        return self.inputs + self.outputs
+
+    def transform(self):
+        pass
```
Comment (jackwish) [inline tflite2onnx/op/logistic.py:41]: GitHub warns  _No newline at the end of file_, it seems to me some editors may contribute this :)

**Review by jackwish — CHANGES_REQUESTED** [review]
Hi @IkbeomJeon thank you for contributing these operators! Finally, I won't write the code all by my own, that means a lot!

I go through this PR quickly and leave several comments. In general
* We spare efforts to keep this repo small enough, the model files is the most significant
  * We plan to track end-to-end models in [`more-model-test branch`](https://github.com/jackwish/tflite2onnx/tree/more-model-test) branch, maybe we can exclude the network model from this PR?
  * The tensor size of the operator can be small. This requires to re-generate yet another TFLite model per operator, yes, I know this could be boring.
* We let CI track our development to ease the maintenance effort
  * Every TFLite operator should have its own unit test, i.e. dedicated TFLite model that contains the operator only. Could you help to have one for Sigmoid?
  * Code style is guarded by `Flake8`, just type `flake8`, it will tell you where to fix.

I know our guide is out of date, I have a plan to fix it this or next week. Thank you again for your effort!

**Comment by jackwish** [comment]
I have changed the title to better demonstrate what is in this PR. Feel free to change into any format you like.

File: tflite2onnx/op/logistic.py [hunk 6]
```diff
@@ -0,0 +1,41 @@
+import logging
```
Comment (jackwish) [inline tflite2onnx/op/logistic.py:1]: Btw, we can include activation operators that don't have attributes in one converter class in https://github.com/jackwish/tflite2onnx/blob/master/tflite2onnx/op/activation.py,  but let me handle that.

**Comment by IkbeomJeon** [comment]
Okay, I'll try as soon as possible. :)

**Comment by jackwish** [comment]
@IkbeomJeon I have merged your change with the latest master branch. I was planning to push to your `add_op` branch, but it seems I am not able to push. Did you enable the _allow modify of maintainer_ option for this PR? Alternatively, I pushed to my [`deconv`](https://github.com/jackwish/tflite2onnx/tree/deconv) branch, you can pull and merge to your `add_op` branch.

There must be something wrong with padding handling in the current code since my tiny model is failing while a `stride == 2` can pass. We can continue to work on it.

**Comment by jackwish** [comment]
I have reworked the code:
* Drop operators that have been enabled
* Move `TransposeConv` to `conv.py`
* Drop paths that have not been tested
* Code style fix

After the test pass, I will merge this PR.

**Comment by jackwish** [comment]
TODO of `TransposeConv` (due to we do not fully understand the semantic gap)
1. Special output shape for VALID padding
2. Different input/output shape for SAME padding

**Review by jackwish — APPROVED** [review]
The test passes, merge as initial support. TODO tracked by https://github.com/jackwish/tflite2onnx/issues/26#issuecomment-731760320

**Pull request merged**